services/convert/pull_review.go

```diff
@@ -66,7 +66,7 @@ func ToPullReviewList(ctx context.Context, rl []*issues_model.Review, doer *user
 	result := make([]*api.PullReview, 0, len(rl))
 	for i := range rl {
 		// show pending reviews only for the user who created them
-		if rl[i].Type == issues_model.ReviewTypePending && !(doer.IsAdmin || doer.ID == rl[i].ReviewerID) {
+		if rl[i].Type == issues_model.ReviewTypePending && !(doer != nil && (doer.IsAdmin || doer.ID == rl[i].ReviewerID)) {
```

Comment:
```suggestion
		if rl[i].Type == issues_model.ReviewTypePending && (doer == nil || (!doer.IsAdmin && doer.ID != rl[i].ReviewerID)) {
```

Reply:
I think the new logic is easier to read:

if Pending AND (NoDoer OR (DoerNotAdmin && DoerNotReviewer))  then continue